charts/seed-bootstrap/templates/extensions/crd-containerruntime.yaml

```diff
@@ -0,0 +1,122 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: containerruntimes.extensions.gardener.cloud
+spec:
+  group: containerruntimes.gardener.cloud
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    plural: containerruntimes
+    singular: containerruntime
+    kind: ContainerRuntime
+    shortNames:
+      - cre
```

Comment:
Is this really needed? `ContainerRuntime` is esoteric enough that there's no point in creating a short name for it.

Reply:
Removed